lib/node_modules/@stdlib/blas/base/srotg/docs/types/index.d.ts

```diff
@@ -0,0 +1,91 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/// <reference types="@stdlib/types"/>
+
+import { Collection } from '@stdlib/types/object';
+
+/**
+ * Inteface describing `srotg`.
+ */
+interface Srotg {
+	/**
+	* Constructs a Givens plane rotation.
+	*
+	* @param a - first value
+    * @param b - second value
+	* @returns output array
+	*
+	* @example
+	* var out = srotg( 0.0, 2.0 );
+	* // returns [ 2.0, 1.0, 0.0, 1.0 ]
+	*
+	* @example
+	* var out = srotg( 6.0, -8.0 );;
+	* // returns [ 10.0, -1.666, -0.6, 0.8 ]
+	*/
+	( a: number, b: number ): Array<number>;
+
+    /**
+    * Constructs a Givens plane rotation.
+    *
+    * @param {number} a - first value
```

Comment:
In TypeScript files, we don't include `{<type>}` annotations as the dtypes are present in the signatures.